Package.swift

```diff
@@ -340,7 +340,7 @@ let package = Package(
         .library(name: "AWSXRay", targets: ["AWSXRay"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/awslabs/smithy-swift", branch: "main"),
+        .package(url: "https://github.com/awslabs/smithy-swift", branch: "jbe/acceptors"),
```

Comment:
The smithy PR https://github.com/awslabs/smithy-swift/pull/483 must merge along with this one, and this must be reverted to `main` prior to this PR merging

Reply:
This has been pointed to `feat/waiters` since https://github.com/awslabs/smithy-swift/pull/483 has merged to that branch.